File: src/main/java/ru/hubsmc/hubscore/module/donate/commands/DonateCommand.java
package ru.hubsmc.hubscore.module.donate.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;
import ru.hubsmc.hubscore.HubsCommand;

import java.util.List;

public class DonateCommand extends HubsCommand {

    public DonateCommand() {
        super("donate", null, true, 0);
    }

    @Override
    public boolean onHubsCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, @NotNull String[] args) {
        return false;
    }

    @Override
    public List<String> onHubsComplete(@NotNull CommandSender sender, @NotNull Command command, @NotNull String alias, @NotNull String[] args) {
        return null;
    }

}

File: src/main/java/su/hubs/hubscore/module/donate/commands/DonateCommand.kt
package su.hubs.hubscore.module.donate.commands

import org.bukkit.command.Command
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player
import su.hubs.hubscore.HubsCommand
import su.hubs.hubscore.module.chesterton.internal.parser.MenuParser
import su.hubs.hubscore.util.PlayerUtils

class DonateCommand : HubsCommand("donate", null, true, 0) {

    override fun onHubsCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        PlayerUtils.openMenuToPlayer(sender as Player, MenuParser.parseChestMenu("donate"))
        return true
    }

    override fun onHubsComplete(sender: CommandSender, command: Command, alias: String, args: Array<String>) = null
}